bench.sh

```diff
@@ -56,6 +57,7 @@ for benchmark in ${BENCHMARKS_TO_RUN}; do
 		--rm \
 		--cpus "${GRPC_SERVER_CPUS}" \
 		--memory "${GRPC_SERVER_RAM}" \
+		--ulimit memlock="${GRPC_MEMLOCK_ULIMIT}" \
```

Comment:
I've introduced a new environment variable GRPC_MEMLOCK_ULIMIT to customize the 'memlock' ulimit setting for Docker containers.
This is because GMF uses the [Glommio](https://docs.rs/glommio/latest/glommio/#before-using-glommio) library, which requires at least 512 KiB of locked memory for io_uring to work

Reply:
Did you notice it was below 512 KiB? On my machine, the default is 8192 KB.

Reply:
You're correct, on a typical standalone system, the default memlock limit is often much higher than 512 KiB.

However, I encountered a different scenario in the CI pipeline environment where resources are often more constrained. In those CI environments, like GitHub Actions or other similar CI/CD platforms, default resource limits can be set much lower to optimize resource usage across multiple running tasks. I've found that in such environments, the memlock limit can indeed fall below the 512 KiB that Glommio requires for io_uring to work correctly

Reply:
Are you sure that it's in bytes and not kilobytes?